recipes/fltk/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "fltk" %}
+{% set version = "1.3.4" %}
+{% set longversion = "1.3.4-2" %}
+{% set md5 = "69128be72365cbedd1d380739f0164e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ longversion }}-source.tar.gz
+  url: http://fltk.org/pub/{{ name }}/{{ version }}/{{ name }}-{{ longversion }}-source.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+
```

Comment:
regarding the log from the windows build:
```
cannot find system zlib library - using built-in
-- 
cannot find system jpeg library - using built-in
-- 
cannot find system png library - using built-in
```
these libraries could be added as host and run-dependencies:

```
  host:
    - zlib
    - jpeg
    - libpng
  run:
    - zlib
    - jpeg
    - libpng
```